### PR TITLE
Like

### PR DESCRIPTION
**Pull request by JuvitaSaini09**
-added post and delete api

files to be checked:
-src/components/singleVideo/singleVideo.js
-src/context/likedVideoContext/likedVideoContext.js
-src/context/singleVideoContext/singleVideoContext.js
-src/util/apiCall.js

### CODE REVIEW

File: src/context/likedVideoContext/likedVideoContext.js [hunk 1]
```diff
@@ -2,29 +2,16 @@ import React, { useContext, createContext, useReducer } from "react";
...
-  return state.filter((item) => item._id !== video._id);
-};
 const LikedVideoProvider = ({ children }) => {
   //<--------reducer function------>
   const likedVideoReducer = (likedVideoState, action) => {
     switch (action.type) {
       case "like":
-        return likedVideoHandler(likedVideoState, action.payload);
+        return action.payload;
+      // return likedVideoHandler(likedVideoState, action.payload);
```
Comment (gauravsinhaweb) [inline src/context/likedVideoContext/likedVideoContext.js:11]: remove console logs

**Review by gauravsinhaweb — APPROVED** [review]
you are good to merge

**Review by neetukumari4858 — APPROVED** [review]
overall looks good 
code is readable
@JuvitaSaini09  good job

**Review by Muzammil-khan710 — APPROVED** [review]
Good to merge Juvita !

**Pull request merged**